main.py

```diff
@@ -0,0 +1,94 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/")
+def root() -> dict:
+    return {"message": "Hello world!"}
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(
+        db: Session = Depends(get_db),
+        skip: int = 0,
+        limit: int = 1
+):
+    return crud.get_all_authors(db=db, skip=skip, limit=limit)
+
+
+@app.get("/authors/{author_id}/", response_model=schemas.Author)
+def read_single_author(author_id: int, db: Session = Depends(get_db)):
+    db_author = crud.get_author_by_id(db=db, author_id=author_id)
+
+    if db_author is None:
+        raise HTTPException(
+            status_code=400,
+            detail="Author with current id none exist"
```

Comment:
The error message contains a typo. It should be 'Author with the current id does not exist'.